example-card-integration/data-connector/actions/importaction.ts

```diff
@@ -8,9 +8,7 @@ export const importAction: (action: DataConnectorAsynchronousAction) => Promise<
     action,
 ) => {
     const primaryKeys = (action.data as any).taskIds as string[];
-    const itemsToAdd: ExampleTaskType[] = getTaskData(primaryKeys).filter((item) =>
```

Comment:
The `getTaskData` function returns the task data for the provided `primaryKeys` so the filter is not required.